EIPS/eip-7733.md

```diff
@@ -0,0 +1,232 @@
+---
+eip: 7733
+title: overflow protection
```

Comment:
title is ambiguous , your current description suits as a better title

Reply:
We already changed the title to zk-SNARK public inputs overflow protection,thanks for your advice